microsoft-edge/devtools-guide-chromium/memory-problems/dom-leaks.md

```diff
@@ -1,37 +1,52 @@
 ---
-title: Debug DOM memory leaks with the Detached Elements tool
-description: Use the Detached Elements tool in the Microsoft Edge DevTools to find and fix DOM memory leaks.
+title: Debug DOM memory leaks with the Detached Elements feature
+description: Find and fix DOM memory leaks by using the Detached Elements profiling type in the Memory tool in Microsoft Edge DevTools.
 author: MSEdgeTeam
 ms.author: msedgedevrel
 ms.topic: conceptual
 ms.service: microsoft-edge
 ms.subservice: devtools
-ms.date: 09/18/2023
+ms.date: 10/13/2024
 ---
-# Debug DOM memory leaks with the Detached Elements tool
+# Debug DOM memory leaks with the Detached Elements feature
 
-> [!IMPORTANT]
-> The **Detached Elements** tool is being deprecated.  Starting with Microsoft Edge 130, the **Detached Elements** tool has a message stating that the tool is deprecated; instead, in the **Memory** tool, in the initial **Select profiling type** screen, select the **Detached elements** option button.  The Edge-only Chrome DevTools Protocol (CDP) function `EdgeDOMMemory.getDetachedNodesIds` continues to work, but use `DOM.getDetachedDomNodes` instead.
-> 
-> In Microsoft Edge 133, the **Detached Elements** tool will be removed; instead, in the **Memory** tool, in the initial **Select profiling type** screen, select the **Detached elements** option button.  The CDP function `EdgeDOMMemory.getDetachedNodesIds` will be removed; use `DOM.getDetachedDomNodes` instead.
+<!-- article covers both tools during Oct 17, 2024 - Jan 15, 2025, then remove Detached Elements tool content & tabset -->
 
-<!-- todo:  
-When the **Detached Elements** tool is removed, delete the present article, redirect to new url:
-Add new article in Memory tool toc bucket: "Debug detached DOM element memory leaks with the Memory tool"
-or possibly reuse present .md file / url; change title & content
--->
+<!-- todo: global: red box on DE tab and Mem tab -->
 
-Use the **Detached Elements** tool to find detached elements that the browser cannot garbage-collect, and locate the JavaScript object that is still referencing the detached element.  By changing your JavaScript to release the element, you reduce the number of detached elements on your page.
+The **Detached Elements** feature finds and displays all of the detached elements on a webpage.
```

Comment:
pls tech-review & massage the technical explanations at top of article
@captainbrosset 

https://review.learn.microsoft.com/microsoft-edge/devtools-guide-chromium/memory-problems/dom-leaks?branch=pr-en-us-3282

Reply:
closing.  non-blocking.
added explanation:
`#### Run GC before getting detached elements, to show only elements that can't be GC'd`
to be further integrated.